libs/ipfs-http-client/Makefile

```
@@ -0,0 +1,74 @@
+# 
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipfs-http-client
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Leonid Esman <leonid.esman@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/vasild/cpp-ipfs-http-client.git
+PKG_SOURCE_DATE:=2019-11-05
+PKG_SOURCE_VERSION:=763e59ad698f3e3846f85df11e01c18ef3fbc401
+PKG_MIRROR_HASH:=a2b5721faf0d43ddb4a892245ef382666149c83f3f97e558e1a6acf2402fb9fd
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)/Default/description
```

```suggestion
define Package/ipfs-http-client/Default/description
```

See for more details https://github.com/openwrt/packages/pull/4119